pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -19,16 +19,35 @@ package plugins
 import (
 	"fmt"
 
+	corelisters "k8s.io/client-go/listers/core/v1"
+	storagelistersv1 "k8s.io/client-go/listers/storage/v1"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm"
 	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/tainttoleration"
 	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	internalcache "k8s.io/kubernetes/pkg/scheduler/internal/cache"
+	"k8s.io/kubernetes/pkg/scheduler/volumebinder"
 )
 
+// DefaultRegistryArgs arguments needed to create default plugin factories.
+type DefaultRegistryArgs struct {
```

Comment:
I prefer to name it to `RegistryArgs`, because when declaring a `DefaultRegistryArgs` variable does not really give you a default registry args by default.

Reply:
This is args for default registry, not default RegistryArgs, but I see how this can be interpreted the other way, I changed it as you suggested.